docs/deployer/helm.md

```diff
@@ -43,6 +60,22 @@ spec:
 
     updateStrategy: update | patch # optional; defaults to update
 
+    # Configuration of the health checks for the resources.
+    # optional
+    healthChecks:
+      # Allows to disable the default health checks.
+      # optional; set to false by default.
+      disableDefault: true
+      # Defines the time to wait before giving up on a resource
+      # to be healthy. Should be changed with long startup time pods.
+      # optional; default to 60 seconds.
```

Comment:
not sure if a default of 1 minute is enough.
I would be more conservative as things could take more time of the image needs to be downloaded first.

But not sure what would be a good default.
maybe 3minutes. but also just a magic number

Reply:
Default is now at 3 minutes for both, HealthChecks and Deletion. Three minutes should be enough for simple cases, if a longer timeout is needed, BluePrint developers needs to change the value in their DeployExecutions to meaningful values that suits their component.